terraform/cp4auto/outputs.tf

```diff
@@ -17,6 +17,25 @@ output "kubeconfig" {
   value = data.ibm_container_cluster_config.cluster_config.config_file_path
 }
 
+output "namespace" {
+  value = var.namespace
+}
+
+output "cp4ba_url" {
+  description = "Use this URL to access your Cloud Pak for Business Automation"
+  value       = module.cp4auto.cp4auto_endpoint
+}
+
+output "cp4ba_username" {
+  description = "Username for your Cloud Pak for Business Automation"
+  value = module.cp4auto.cp4auto_username
+}
+
+//output "cp4ba_password" {
```

Comment:
Is there a reason the password block has been commented out?

Reply:
Not really. Since it's password, I guess it's better to keep it confidential. Plus, the user may well remember what he or she enters during the process.